blogs/packet_drops/packets_drop.md

```diff
@@ -53,9 +51,85 @@ collector object with the following fields enabled in eBPF config
 section
 
 ```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
   agent:
     type: EBPF
     ebpf:
       privileged: true
-      enablePktDrop: true
+      features:
+        - PacketsDrop
```

Comment:
That should end like this:
https://github.com/netobserv/network-observability-operator/pull/394/files#r1304927133

Let's keep an eye on the final output